src/internal/collection/types.go

```diff
@@ -21,7 +21,7 @@ type PostgresCollection interface {
 	// invalidated at the end of the transaction.  Basically, it's
 	// software transactional memory.  See this blog post for details:
 	// https://coreos.com/blog/transactional-memory-with-etcd3.html
-	ReadWrite(tx *pachsql.Tx) PostgresReadWriteCollection
+	ReadWrite(ctx context.Context, tx *pachsql.Tx) PostgresReadWriteCollection
```

Comment:
Since no actual work happens in `ReadWrite` and the context would be unused if not stashed in a struct, you may omit the context if you like.